script/main.py

```diff
@@ -19,10 +25,160 @@ def init_argparser():
   return args
 
 
-def init_db():  
-  '''Initializing the database'''
+def create_connection(db_file):
+  conn = None
+  try:
+    conn = sqlite3.connect(db_file)
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+def create_table(conn, create_table_sql):
+  try:
+    cursor = conn.cursor()
+    cursor.execute(create_table_sql)
+  except sqlite3.Error as e:
+    print(e)  
+
+
+def insert_users_to_table(conn, users):
+  sql = '''INSERT INTO users VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)'''
+
+  cur = conn.cursor()
+  for columns in users:
+    cur.execute(sql, columns)  
+  conn.commit()
 
-  print('Nothing is implemented')
+
+def init_db(conn):  
+  '''Initializing the database'''  
+
+  
+  sql_create_users_table = ''' CREATE TABLE IF NOT EXISTS users (
+    gender text,
+    name_title text,
+    name_first text,
+    name_last text,
+    location_street_number integer,
+    location_street_name text,
+    location_city text,
+    location_state text,
+    location_country text,
+    location_postcode integer,
+    location_coordinates_latitude numeric,
+    location_coordinates_longitude numeric,
+    location_timezone_offset numeric,
+    location_timezone_description text,
+    email text,
+    login_uuid text,
+    login_username text,
+    login_password text,
+    login_salt text,
+    login_md5 text,
+    login_sha1 text,
+    login_sha256 text,
+    dob_date text,
+    dob_age text,
+    registered_date text,
+    registered_age text,
+    phone text,
+    cell text,
+    id_name text,
+    id_value text,
+    picture_large text,
+    picture_medium text,
+    picture_thumbnail text,
+    nat
+  ); '''  
+
+  # create table
+  if conn is not None:
+    create_table(conn, sql_create_users_table)
+  else:
+    print("Error! cannot create the database connection.")
+  
+
+def import_users_to_db(conn):
+  people = []
+
+  list_of_people = unflat_people['results']
```

Comment:
unflat_people (why this name?) should be passed as an argument here not as global variable